src/makefile

```diff
@@ -271,24 +253,6 @@ $(sort $(patsubst %.cc,%.ii,$(CXXUSERSRCS))): %.ii: %.cc
 	$(ECHO) Preprocessing $< to $@
 	$(Q)$(CXX) -dD $(CXXFLAGS) $(EXTRAFLAGS) -E $< -o $@
 
-ifeq ($(TRIVIAL_BUILD),no)
-configure: configure.ac
```

Comment:
Can you please let me know why you chose to remove this block, which helps re-generate and re-run configure when configure.ac is modified? This feels like a regression, since the user now has to manually re-run autogen.sh

Reply:
There are a few reasons:
- `autogen.sh` is the next file to be removed as it doesn't make sense to keep an extra wrapper around the `autoreconf`,
- if you run `autoreconf` then the generated `Makefile` will contains this logic - however, it will become usable once the existing logic from the `makefile` will be moved to the `Makefile.am` then the `makefile` could be entirely removed - at that time the `LinuxCNC` project will become more `autotool` compliant.
- this is not a crucial functionality - so it should be acceptable to document that you need to run `autoreconf` after `configure.ac` was modified - which you need to run anyway if you start compiling the project from the scratch.